addons/sms/models/sms_sms.py

```diff
@@ -58,6 +58,34 @@ def send(self, delete_all=False, auto_commit=False, raise_exception=False):
             if auto_commit is True and not getattr(threading.currentThread(), 'testing', False):
                 self._cr.commit()
 
+    def retry(self):
```

Comment:
Just for guidelining, it should be called action_set_outgoing (action to indicate this is used by form view button, set_outgoing as actually retry is a side effect).

Reply:
Dev Done !